cocktails9/cocktails9/Views/login.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+
+struct login: View {
+    
+    @State private var currentImage = "loginbw2"
+    @State private var email = ""
+    @State private var password = ""
+    @State private var showAlert = false
+    @State private var alertMessage = ""
+    @State private var navigateToGrid = false
+    
+    var body: some View {
+        NavigationView {
+            VStack(spacing: 15) {
+                Spacer(minLength: 0)
+                
+                HStack {
+                    Text("Lo")
+                        .font(.largeTitle)
+                        .fontWeight(.bold)
+                        .padding(.trailing, -7.0)
```

Comment:
`HStack` has a `spacing` param, just like `VStack`. So you can remove this negative padding from this Text.
```suggestion
                HStack(spacing: .zero) {
                    Text("Lo")
                        .font(.largeTitle)
                        .fontWeight(.bold)
```